benchmarks/shelley3pools/create-addresses-generic.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+. configuration/parameters
+
+ID=$1
+
+NUM_OF_ADDRESSES=${NUM_OF_ADDRESSES:-3}
+CLICMD="../../bin/cardano-cli"
+
+WORKDIR=./tmp
```

Comment:
this might serve as the temporary directory? also placing configurations and topologies here?